drivers/sps30/sps30_saul.c

```diff
@@ -65,7 +65,7 @@ static void _float_fit(float *src, phydat_t *data, size_t dim, uint32_t mul)
     phydat_fit(data, &i32[0], dim);
 }
 
-static int read(const void *dev, phydat_t *data, unsigned int val_idx)
+static int do_read(const void *dev, phydat_t *data, unsigned int val_idx)
```

Comment:
In this file you have other internal functions that have prefix `_`. Here it would make sense to use `_read` for internal read in the `read_*` functions.

Reply:
good idea